mmv1/products/filestore/api.yaml

```diff
@@ -144,6 +144,12 @@ objects:
                 File share capacity in GiB. This must be at least 1024 GiB
                 for the standard tier, or 2560 GiB for the premium tier.
               required: true
+            - !ruby/object:Api::Type::String
```

Comment:
Is adding this field part of the original request? We should not add it if not. Also, this is untested in this PR.

Reply:
It is an output field read onlly of that resource, I figured it should be supported with the backup implemented.

Reply:
Is it? You did not mark it as such.

Reply:
> > @googlerjk The latest commit does not seem to address all the comments @c2thorn left. Is this the latest version? Sorry if I have missed anything. Comments that need to be addressed:
> > 
> > 1. what's the purpose of modifying example at [mmv1/templates/terraform/examples/filestore_instance_full.tf.erb](https://github.com/GoogleCloudPlatform/magic-modules/pull/6742/files/6515acb233f76651569eba9bc379c5233ccc35bf#diff-ace55e9c3d3c953f02941d02cce5c760d163213e097d67482508796d363a9231)? The goal of that example is to test the `google_filestore_instance` resource
> > 2. `input: true` is added to `nfsExportOptions` in `google_filestore_instance` resource? Is that intentional?
> 
> RE: #1 since its the "full" version of the filstore, I included backup in it. And this is why I made the changes. RE: #2 looks like my update didn't get pushed. The field should be set to input. I'll push that now.

RE: #1
I would still suggest we remove the google_filestore_backup part from filestore_instance_full test, as the newly added part tests the exactly same functionality of the `google_filestore_backup` resource as in filestore_backup_basic test. It would not be necessary to test the same functionality in two tests.

Reply:
I tested it locally and it doesn't seem the backup name is returned for this field. Filestore instance and backup got created successfully though. Do you happen to know why it's not got returned?